Wikipedia/Categories/UIView+WMFDefaultNib.m

```diff
@@ -1,17 +1,15 @@
-//
-//  UIView+WMFDefaultNib.m
-//  Wikipedia
-//
-//  Created by Brian Gerstle on 3/6/15.
-//  Copyright (c) 2015 Wikimedia Foundation. All rights reserved.
-//
 
 #import "UIView+WMFDefaultNib.h"
+#import "NSString+Extras.h"
 
 @implementation UIView (WMFDefaultNib)
 
 + (NSString*)wmf_nibName {
-    return NSStringFromClass(self);
+    /* Swift has "Namespaced" class names that prepend the module
```

Comment:
there's a ticket for Swift naming conventions, in which I propose we have `@objc` name-spaced names for classes that are used in objc. For example, `WMFImageController becomes `@objc(WMFImageController) ImageController`.  Doing this with cell classes would allow them to be referenced via prefix in ObjC and obviate these kinds of workarounds.  My only reservations with this are:

- Performance (very, very minor, but this is unnecessary in the vast majority of cases in the app)
- [Name mangling](http://stackoverflow.com/questions/24321773/how-can-i-demangle-a-swift-class-name-dynamically) in Swift could make this even more problematic
- Nested types (YAGNI, but possible down the line)

For example:
``` swift
@objc(WMFMyCell) public class MyCell : UITableViewCell {
}
```

results in:

```
(lldb) po NSStringFromClass([WMFMyCell class])
WMFMyCell
(lldb) po [WMFMyCell wmf_nibName]
WMFMyCell
```